numpy/core/src/umath/ufunc_object.c

```diff
@@ -5396,6 +5396,8 @@ ufunc_outer(PyUFuncObject *ufunc, PyObject *args, PyObject *kwds)
     PyArrayObject *ap1 = NULL, *ap2 = NULL, *ap_new = NULL;
     PyObject *new_args, *tmp;
     PyObject *shape1, *shape2, *newshape;
+    PyObject *numpy = PyImport_ImportModule("numpy");
+    PyObject *numpy_matrix = PyObject_GetAttrString(numpy, "matrix");
```

Comment:
I should probably use `npy_cache_import` here

Reply:
And I guess by I, I mean you

Reply:
Cool, I didn't know about that. Done now, I think.